database/buildSchema/44_data_display.sql

```diff
@@ -29,3 +29,12 @@ CREATE TABLE data_view_column_definition (
     UNIQUE (table_name, column_name)
 );
 
+-- Table for cataloguing all "data" tables, including lookup tables
```

Comment:
Please rename this file to `43_data_table` if possible.

Reply:
You mean 'data_view'? 😜

Reply:
Done.